### PR TITLE
[SC] Hot-patch Reaction and Stitch to test a fix for the CSS bloat.

### PR DESCRIPTION
**Pull request by alloy**
This PR is to test a possible fix for https://artsyproduct.atlassian.net/browse/PLATFORM-1184. Because it's hard to reproduce this locally, I had to rely on other indicators, which indicate this issue should be fixed. However, the proof is in the pudding, as they say, so I don’t want to call this done until we've seen it fixed in staging/production.

The fix spans a few repos, though, so rather than doing a bunch of releases that perhaps are deemed unnecessary, after testing this fix, I went with patched versions of these packages. The corresponding upstream PRs are:

* https://github.com/artsy/reaction/pull/2029
* https://github.com/artsy/stitch/pull/61

In short, this PR changes the Reaction integration to expect rendered HTML from Reaction and use that for the page’s body as-is.

### CODE REVIEW

**Comment by alloy** [comment]
FWIW I tested this locally against all the types of pages that use Reaction apps, but testing them on staging would be prudent.

File: patches/@artsy/stitch+6.0.0.patch [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+\ No newline at end of file
```
Comment (damassi) [inline patches/@artsy/stitch+6.0.0.patch]: Patch package FTW

File: src/desktop/apps/artist/server.tsx [hunk 2]
```diff
@@ -17,21 +17,21 @@ app.get(
       const user = req.user && req.user.toJSON()
       const { ARTIST_INSIGHTS } = res.locals.sd
       const {
-        ServerApp,
+        bodyHTML,
```
Comment (damassi) [inline src/desktop/apps/artist/server.tsx]: This is probably how it should have been done from the beginning. I was envisioning being able to pass props along from consuming apps (before rendering on the server) but in practice we haven't needed to do this.

**Comment by damassi** [comment]
merge on green

File: patches/@artsy/stitch+6.0.0.patch (hunk 1 above)
Comment (alloy) [reply patches/@artsy/stitch+6.0.0.patch]: I know, right? I absolutely ❤️ it. This package speaks my language.

File: src/desktop/apps/artist/server.tsx (hunk 2 above)
Comment (alloy) [reply src/desktop/apps/artist/server.tsx]: Yeah I can see how you might think that, but now we know we don’t need it for sure and can simplify 👍
